telegram/ext/jobqueue.py

```diff
@@ -21,10 +21,22 @@
 import logging
 import time
 import warnings
+import datetime
+from enum import Enum
```

Comment:
I agree to use enum because Python 2 is going to drop its support soon. Python 3 is the new standard.